docs/proto/proto-docs.md

```diff
@@ -304,21 +305,8 @@ GenesisState - initial state of module
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
 | `params` | [Params](#confio.poe.v1beta1.Params) |  | params defines all the parameter of the module |
-| `seed_contracts` | [bool](#bool) |  | SeedContracts when enabled stores and instantiates the Proof of Engagement contracts on the chain. |
-| `gen_txs` | [bytes](#bytes) | repeated | GenTxs defines the genesis transactions to create a validator. |
-| `system_admin_address` | [string](#string) |  | SystemAdminAddress single address that is set as admin for the PoE contracts in seed mode. |
 | `contracts` | [PoEContract](#confio.poe.v1beta1.PoEContract) | repeated | Contracts Poe contract addresses and types when used with state dump in non seed mode. |
```

Comment:
Since all the "Seed Mode" is moved into it's own key (I agree), I want to check two things:

1. Is `contracts` all we will ever need in non-seed mode? (If not, let's make another sub-structure for it)
2. If this is all we will use, is there a way to rename it, to show this is only for importing existing chains (ignored on initial "clean" genesis)?

Reply:
A `oneof` element would be cleanest IMHO as it separates the two use cases.  I still need to find out if we need to export historical informations. I have also asked on IBC discord now.